### PR TITLE
Use Sass for the majority of the CSS

### PR DESCRIPTION
**Pull request by jonathantneal**

### CODE REVIEW

**Comment by jonathantneal** [comment]
Thanks, @ygorcosta! I’ve assigned the merge to you, @zenorocha. Thanks, everyone. 👍

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: node/magnet/postcss.config.js [hunk 1]
```diff
@@ -79,3 +68,35 @@ const compress = postcss.plugin('postcss-discard-tested-duplicate-declarations',
...
+      // promise the sass-transpiled css as a new ast
+      result => postcss.parse(
+        result.css.toString()
+      )
+    ).then(
+      // replace the old ast with the new ast
+      newcss => css.removeAll().append(...newcss.nodes)
+    );
+  }
+);
```
Comment (zenorocha) [inline node/magnet/postcss.config.js]: This function looks a little bit overcomplicated, can't we simplify it in some way? Maybe compiling sass using npm scripts?

Comment (jonathantneal) [reply node/magnet/postcss.config.js]: Yes, it is overly complicated and I did simplify the function:

https://gist.github.com/jonathantneal/32a0f1044d149837222f3255e9dc5e26#file-postcss-node-sass-js

However, I only needed a Sass PostCSS plugin when I had PostCSS plugins running both before and after it in the chain — it was a necessity as I was stripping away PostCSS in parts. Now I can and will probably make it separate npm tasks.

**Pull request merged**